java/fury-core/src/main/java/io/fury/Fury.java

```diff
@@ -703,12 +703,7 @@ public Object deserialize(InputStream inputStream) {
   public Object deserialize(InputStream inputStream, Iterable<MemoryBuffer> outOfBandBuffers) {
     buffer.readerIndex(0);
```

Comment:
Can we move this line into `transferObjectFromStream`?

Reply:
Done.